api/controllers/auth-controller.js

```diff
@@ -47,7 +48,10 @@ const signIn = async (req, res) => {
         if (!validPassword) return res.status(400).json({ message: 'Invalid email or password.'});
 
         const token = user.generateAuthToken();
-        res.status(200).send(token);
+        const { username } = user;
+        res
+            .header("access-token", token)
+            .status(200).json({user: {email, username}})
```

Comment:
Why do you return only username and email?? You should return all user info except password

Reply:
There is only one more property in user  it's id, do we need to add this too? because in our situation we use just email, and username